test/integration/narrow/app/fuzzySearchService.test.js

```diff
@@ -217,4 +217,24 @@ describe('fuzzySearchService tests with filterBy', () => {
       expect(matchingSet.scheme).toBe(schemes[0])
     })
   })
+
+  test('GET /paymentdata filters by single county', async () => {
+    const counties = ['Cambridgeshire']
+    const filteredResult = await getPaymentData({ ...searchCriteria, filterBy: { counties } })
+
+    filteredResult.rows.forEach(x => {
+      const matchingSet = paymentestdata.find(td => td.payee_name === x.payee_name && td.part_postcode === x.part_postcode)
+      expect(matchingSet.county_council).toBe(counties[0])
+    })
+  })
+
+  test('GET /paymentdata filters by multiple counties', async () => {
+    const counties = ['Cambridgeshire', 'Staffordshire']
+    const filteredResultPage = await getPaymentData({ ...searchCriteria, filterBy: { counties } })
+
+    filteredResultPage.rows.forEach(x => {
+      const matchingSet = paymentestdata.find(td => td.payee_name === x.payee_name && td.part_postcode === x.part_postcode)
+      expect(counties.includes(matchingSet.county_council)).toBeTruthy()
+    })
+  })
```

Comment:
Please add another test which checks that counties filter works with other filters such as scheme or amounts

Reply:
ok, I have added the test cases and pushed the code now.